Vagrantfile

```diff
@@ -6,72 +6,23 @@
 # backwards compatibility). Please don't change it unless you know what
 # you're doing.
 
+$script = <<SCRIPT
+cd /home/vagrant/oppia
+apt-get update && apt-get install -y python-pip python-dev unzip
+bash ./scripts/setup.sh
+bash ./scripts/install_third_party.sh
+SCRIPT
```

Comment:
@bbriggs I'd say we should ask once,  as this script can also be run on non VM installations and before we elevate ourselves to admin and install *all the things*  what we should probably ask politely. 

your thoughts?